Assets/Scripts/UI/Canvas/PhotoTaking/CameraPanel.cs

```diff
@@ -24,5 +25,10 @@ public void SetPanelData(PhotoData targetPhotoData)
             // to do: set render from scriptable object
             taker.StartCamera(background);
         }
+        private void Update()
+        {
+            if(taker.WebcamTexture && taker.WebcamTexture.isPlaying)
+                Graphics.Blit(taker.WebcamTexture, renderTexture);
+        }
```

Comment:
Waarom zie ik niks van de vorige feedback over de `OnEnable` & `OnDisable`. Dit hoort netter te staan.

Reply:
```suggestion
        private void Update()
        {
            WebCamTexture currentWebcamTexture = taker.WebcamTexture;
            
            if(currentWebcamTexture 
                && currentWebcamTexture.isPlaying)
                Graphics.Blit(taker.WebcamTexture, renderTexture);
        }
```
Dit is iets netter.